README.md

```diff
@@ -11,61 +11,61 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 -->
 
-# AdoptOpenJDK Testing
+# adoptium Testing
 
-#### Guide to the Test Jobs at AdoptOpenJDK
+#### Guide to the Test Jobs at adoptium
 
-For nightly and release builds, there are test jobs running as part of the AdoptOpenJDK continuous delivery pipelines.  There is a [blog post and brief presentation](https://blog.adoptopenjdk.net/2017/12/testing-java-help-count-ways) that explains what testing we run and how they fit into the overall delivery pipeline.  As the world of testing at AdoptOpenJDK is evolving and improving quickly, some documentation may fall behind the march of progress.  Please let us know and help us keep it up-to-date, and ask questions at the [AdoptOpenJDK testing Slack channel](https://adoptopenjdk.slack.com/messages/C5219G28G)!
+For nightly and release builds, there are test jobs running as part of the adoptium continuous delivery pipelines.  There is a [blog post and brief presentation](https://blog.adoptium.net/2017/12/testing-java-help-count-ways) that explains what testing we run and how they fit into the overall delivery pipeline.  As the world of testing at adoptium is evolving and improving quickly, some documentation may fall behind the march of progress.  Please let us know and help us keep it up-to-date, and ask questions at the [adoptium testing Slack channel](https://adoptium.slack.com/messages/C5219G28G)!
```

Comment:
https://blog.adoptopenjdk.net/2017/12/testing-java-help-count-wayURL should not be changed